libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -39,11 +39,12 @@ const AP_Param::GroupInfo AP_OSD::var_info[] = {
     // @Param: _TYPE
     // @DisplayName: OSD type
     // @Description: OSD type
-    // @Values: 0:None,1:MAX7456,2:SITL,3:MSP
+    // @Values: 0:None,1:MAX7456,2:SITL,3:MSP,4:TX
```

Comment:
needs a bit more explanation, maybe TXONLY (or CRSF?) and something in the description saying what it is for

Reply:
Fixed